.github/policies/resourceManagement.yml

```diff
@@ -1,94 +1,105 @@
-id: 
+id:

```

Comment:
seems my editor fixed a bunch of the formatting here. Sorry for the confusing diff